frontend/src/app/components/signup/signup.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405405)
```suggestion
import { Component, OnInit } from "@angular/core";
```